OpenRVS/classes/OpenMultiPlayerWidget.uc

```diff
@@ -165,18 +167,35 @@ function GetGSServers()
 		NewItem.Created();
 		NewItem.iMainSvrListIdx = i;
 		NewItem.bFavorite = true;
-		NewItem.bSameVersion = true;
+		NewItem.bSameVersion = !ServerList[i].bWrongVersion;//true;//1.5 change

```

Comment:
Want bSameVersion to default to True, but a new bool defaults to False, so did ! to make the bSameVersion for the list item default true.

Reply:
For clarity maybe it's ok to do this:
```
local bool samever;
samever = true;
NewItem.bSameVersion = samever;
```